Configuration/IsGranted.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sensio\Bundle\FrameworkExtraBundle\Configuration;
+
+/**
+ * The Security class handles the Security annotation.
+ *
+ * @author Ryan Weaver <ryan@knpuniversity.com>
+ * @Annotation
+ */
+class IsGranted extends Security
+{
+    private $attributes = array();
+
+    private $object;
```

Comment:
Should actually be called `subject`. It still specifies `object` in the docs too, but voters already call it a subject: http://api.symfony.com/3.1/Symfony/Component/Security/Core/Authorization/Voter/VoterInterface.html

The fact that the subject is not limited to just objects might help swaying the decision ^^